### PR TITLE
Now supporting multiple filtering databases

### PR DESCRIPTION
**Pull request by charles-cowart**
mg-scripts now supports using multiple human-filtering databases in series.
Changes here reflect changes in mg-script's configuration file.

### CODE REVIEW

**Comment by charles-cowart** [comment]
@antgonza This PR supports https://github.com/biocore/mg-scripts/pull/75 and should pass all tests once 75 is approved and merged into the main repo.

**Review by antgonza — APPROVED** [review]
Looks good, thank you! I'll wait for that other PR to be merged to update the tests here.

**Pull request merged**
